core/bin/external_node/src/main.rs

```diff
@@ -117,6 +131,23 @@ async fn init_tasks(
         config.optional.miniblock_seal_queue_capacity,
     );
     task_handles.push(tokio::spawn(miniblock_sealer.run()));
+    let pool = connection_pool.clone();
+    task_handles.push(tokio::spawn(async move {
+        loop {
+            let protocol_version = pool
+                .access_storage()
+                .await
+                .unwrap()
+                .protocol_versions_dal()
+                .last_used_verion_id()
+                .await
+                .unwrap_or(ProtocolVersionId::Version0);
```

Comment:
Would be cleaner if you report metric only if value is some instead or `unwrap_or `

Reply:
The node may be not syncing and have `0` batches and then it would not report anything though